src/src/Helpers/ESPEasy_math.cpp

```diff
@@ -72,13 +76,13 @@ bool approximatelyEqual(const float& a, const float& b) {
 #if FEATURE_USE_DOUBLE_AS_ESPEASY_RULES_FLOAT_TYPE
 bool approximatelyEqual(const double& a, const double& b, double epsilon)
 {
-  return std::abs(a - b) <= ((std::abs(a) < std::abs(b) ? std::abs(b) : std::abs(a)) * epsilon);
+  return std::abs(a - b) <= epsilon;

```

Comment:
Are you seriously trying to correct Donald E. Knuth?

Well, it is an incorrect fix anyway, as should essentially be comparing against the largest abs. value of both times epsilon.

The idea of epsilon is the smallest value where 1 + epsilon != 1.
So of you compare larger values, you should also 'shift' epsilon by the same order of magnitude.

Simple example:

Just imagine a floating point resolution of a `float`, which is roughly 6 a 7 decimals.
So for values around 1.0 the epsilon should be about 0.0000001 give or take.

Now look at values 1000000 and 1000000.001.
With your changes those are not the same as the difference is 0.001, but if you convert both string representations to a `float` type, you should get something which should be considered as `true` when using `approximatelyEqual` on them.

Now consider 0.0000000001 and 0.0000000002
Those should absolutely not return `true` when using `approximatelyEqual` on them.
However using your changes, they will as the absolute difference is smaller than epsilon.

Reply:
Ah check, now I see what you're trying to fix...
So Knuth is still correct and the factor isn't applied twice :)